sandbox/pages/SandboxTreeList.vue

```diff
@@ -12,28 +12,43 @@
       <SandboxSectionComponent title="item">
         <KTreeList :items="items1" />
       </SandboxSectionComponent>
+      <SandboxSectionComponent title="group">
```

Comment:
Can you update the example to have 2 sets of treelists like on the docs example?

Reply:
Done